composer.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "extend/module-warranty-graph-ql",
+    "name": "helloextend/module-warranty-graph-ql",
     "description": "",
```

Comment:
Can you give a short description of this specific module?  It helps provide something in packagist.

Reply:
done.